src/applications/personalization/dashboard-2/components/apply-for-benefits/ApplicationInProgress.jsx

```diff
@@ -3,48 +3,118 @@ import PropTypes from 'prop-types';
 
 import { recordDashboardClick } from '~/applications/personalization/dashboard/helpers';
 
+/**
+ * Returns a copy of the passed-in string with the first letter capitalized
+ *
+ * @param {string} input - word to capitalize
+ * @returns {string} - the input string with the first letter capitalized
+ */
+const capitalizeFirstLetter = input => {
+  const capitalizedFirstLetter = input[0].toUpperCase();
+  return `${capitalizedFirstLetter}${input.slice(1)}`;
+};
+
 const ApplicationInProgress = ({
   continueUrl,
   expirationDate,
   formId,
   formTitle,
   lastOpenedDate,
   presentableFormId,
+  removeForm,
+  startNewApplicationUrl,
 }) => {
+  const isExpired = React.useMemo(
+    () => new Date(expirationDate).getTime() < Date.now(),
+    [expirationDate],
+  );
+
   return (
     <div
       className="vads-u-display--flex vads-l-col--12 medium-screen:vads-l-col--6 small-desktop-screen:vads-l-col--4 medium-screen:vads-u-padding-right--3 vads-u-padding-bottom--3"
       data-testid="application-in-progress"
     >
       <div className="vads-u-display--flex vads-u-width--full vads-u-flex-direction--column vads-u-justify-content--space-between vads-u-align-items--flex-start vads-u-background-color--gray-lightest vads-u-padding--2p5">
-        <div>
-          <div className="vads-u-text-transform--uppercase">
-            {presentableFormId}
-          </div>
-          <h4 className="vads-u-font-size--h3 vads-u-margin-top--0">
-            {formTitle}
-          </h4>
-          <div className="vads-u-display--flex">
-            <i
-              aria-hidden="true"
-              className={`fas fa-fw fa-exclamation-circle vads-u-margin-right--1 vads-u-margin-top--0p5`}
-            />
+        {!isExpired && (
+          <>
             <div>
-              <p className="vads-u-margin-top--0">
-                Application expires on: {expirationDate}
+              <p className="vads-u-text-transform--uppercase vads-u-margin-y--0">
+                {presentableFormId}
               </p>
-              <p>Last opened on: {lastOpenedDate}</p>
+              <h4 className="vads-u-font-size--h3 vads-u-margin-top--0">
+                {capitalizeFirstLetter(formTitle)}
```

Comment:
Does the design still look good when the `formTitle` is long?

Reply:
Yeah, they are all long 😆  It just wraps to three or four lines but all the cards look nice next to each other.